gap/io.gi

```diff
@@ -908,24 +909,23 @@ function(func, s)
 
   range := range + 1;
   source := source + 1;
-  return DigraphNC(IsMutableDigraph, (rec(DigraphNrVertices := n,
+  return DigraphNC(filt, (rec(DigraphNrVertices := n,
                                       DigraphRange := range,
                                       DigraphSource := source)));
 end);
 
 InstallMethod(DigraphFromSparse6StringCons,
 "for IsImmutableDigraph and a string",
 [IsImmutableDigraph, IsString],
-function(filt, s)
+function(_, s)
   local D;
   D := MakeImmutable(DigraphFromSparse6String(IsMutableDigraph, s));
   SetIsSymmetricDigraph(D, true);
   return D;
 end);
 
 InstallMethod(DigraphFromSparse6String, "for a function and a string",
-[IsFunction, IsString],
-DigraphFromSparse6StringCons);
+[IsFunction, IsString], DigraphFromSparse6StringCons);
```

Comment:
Why?

Reply:
As I think Reimer Behrens once said: "vertical screen real estate is at a premium", so one less line is a good thing from this perspective.